Source/MediaInfo/Multiple/File_Mpeg4.cpp

```diff
@@ -425,7 +426,7 @@ void File_Mpeg4::Streams_Finish()
     if (Retrieve(Stream_General, 0, General_Format)==__T("Final Cut EIA-608"))
     {
         for (streams::iterator Stream=Streams.begin(); Stream!=Streams.end(); ++Stream)
-            for (size_t Pos=0; Pos<Streams[(int32u)Element_Code].Parsers.size(); Pos++)
+            for (size_t Pos=0; Pos<Streams[(int32u)Element_Code].Parsers.size() && Pos<Stream->second.Parsers.size(); Pos++)

```

Comment:
Actually the main issue is the mismatch between `Streams[(int32u)Element_Code]` and `Stream`, I edit this commit in order to remove the mismatch.